elements/campaign-brand-messaging/campaign-brand-messaging.test.js

```diff
@@ -0,0 +1,45 @@
+import { assert } from 'chai';
+import './campaign-brand-messaging';
+import fetchMock from 'fetch-mock';
+
+describe('<campaign-brand-messaging>', function () {
+  let element;
+  let src;
+  let campaign;
+
+  beforeEach(function (done) {
+    fetchMock.restore();
+    src = 'http://example.com/campaign.json';
+    campaign = {
+      id: 1,
+      active: true,
+      clickthrough_url: 'http://example.com/clickthrough',
+      image_url: 'http://example.com/campaign-img.jpg',
+      name: 'Test Campaign',
+      product_shot_url: 'http://example.com/prodcut-shot.jpg',
+      brand_messaging: 'Buy our shit',
+    };
+    fetchMock.mock(src, campaign);
+
+    element = document.createElement('campaign-brand-messaging');
+    element.setAttribute('src', src);
+
+    setImmediate(function () {
+      done();
+    });
+  });
+
+  it('renders an <campaign-brand-messaging>', function () {
+    assert.equal(element.tagName.toLowerCase(), 'campaign-brand-messaging');
```

Comment:
Not sure if this is the pattern elsewhere, but given we're using `expect(...)` syntax elsewhere in this test, we should use that here as well.

```
expect(element.tagName.toLowerCase()).to.equal('campaign-brand-messaging');
```